new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import oracle.weblogic.kubernetes.actions.impl.TraefikParams;
+import oracle.weblogic.kubernetes.actions.impl.primitive.HelmParams;
+import oracle.weblogic.kubernetes.annotations.tags.MustNotRunInParallel;
+import oracle.weblogic.kubernetes.annotations.tags.Slow;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import oracle.weblogic.kubernetes.extensions.Timing;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static oracle.weblogic.kubernetes.actions.TestActions.createIngress;
+import static oracle.weblogic.kubernetes.actions.TestActions.createUniqueNamespace;
+import static oracle.weblogic.kubernetes.actions.TestActions.deleteNamespace;
+import static oracle.weblogic.kubernetes.actions.TestActions.getExpectedResult;
+import static oracle.weblogic.kubernetes.actions.TestActions.installTraefik;
+import static oracle.weblogic.kubernetes.actions.TestActions.uninstallIngress;
+import static oracle.weblogic.kubernetes.actions.TestActions.uninstallTraefik;
+import static oracle.weblogic.kubernetes.actions.TestActions.upgradeTraefik;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.isTraefikReady;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+
+// this is a simple traefik validation test
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Simple validation of Traefik functions")
+@ExtendWith(Timing.class)
+class ItSimpleTraefikValidation implements LoggedTest {
+
+  private HelmParams tfHelmParams = null;
+  private HelmParams ingressParam = null;
+  private String tfNamespace = null;
+  private String domainNamespace1 = null;
+  private String projectRoot = System.getProperty("user.dir");
+
+  @Test
+  @Order(1)
+  @DisplayName("Install the Traefik")
+  @Slow
+  @MustNotRunInParallel
+  public void testInstallingTraefik() {
```

Comment:
as @rjeberhard said - this should not be a test - we are not testing traefik.  we are testing WKO and MII.  this should just be done in a setup method

Reply:
added create mii domain test to verify the load balancer can route the requests to the cluster